vowpal_wabbit/centos7_6_1860-build.Dockerfile

```diff
@@ -0,0 +1,76 @@
+# tl;dr
+# (docker needs to be configured with 16+ cores and 32GB ram or change the -j16)
+# * docker build - < centos7_6_1860-build.Dockerfile
+# (replace 27c82a691b62 with whatever got printed at the end of docker build)
+# * docker tag 27c82a691b62 eisber/centos7.6.1810-build:0.1.0
+# * docker push eisber/centos7.6.1810-build
+
+FROM centos:centos7.6.1810
+
+RUN yum -y updateinfo \
+   && yum install -y \
+   # C++ to bootstrap
+   gcc-c++ gcc git make wget help2man \
+   # Java
+   maven \
+   # Dependencies
+   boost-static glibc-static \
+   # gcc-9.2 pre-requisites
+   bzip2 bison yacc flex \
+   # Java
+   java-11-openjdk
+
+# Install CMake 3.13
+RUN version=3.13 && build=5 \
+   && mkdir ~/temp-cmake \
+   && cd ~/temp-cmake \
+   && wget https://cmake.org/files/v$version/cmake-$version.$build-Linux-x86_64.sh \
+   && mkdir /opt/cmake \
+   && sh cmake-$version.$build-Linux-x86_64.sh --prefix=/opt/cmake --skip-license \
+   && ln -s /opt/cmake/bin/cmake /usr/bin/cmake \
+   && rm -f cmake-$version.$build-Linux-x86_64.sh
+
+# Install gcc-9.2
+RUN wget ftp://ftp.mirrorservice.org/sites/sourceware.org/pub/gcc/releases/gcc-9.2.0/gcc-9.2.0.tar.gz \
+   && tar xfz gcc-9.2.0.tar.gz \
+   && cd gcc-9.2.0 \
+   && ./contrib/download_prerequisites \
+   && ./configure --disable-multilib --enable-languages=c,c++ --with-pic \
+   && make -j 16 \
```

Comment:
Can you make this parallelization generic? I dont think it will work on a build machine like this